__tests__/app.test.js

```diff
@@ -30,3 +34,23 @@ describe("GET /api/items", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test.only("Responds with a single object, which should have the following properties: author, title, article_id, body, topic, created_at and votes", () => {
+    const article_id = 3;
+    return request(app)
+      .get(`/api/articles/${article_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article[0]).toEqual({
```

Comment:
Because it's only one article, so  expect what comes back is an object instead of the array